patterns/2-structured/project-setup/release-process.md

```diff
@@ -0,0 +1,47 @@
+## Title
+
+Standard Release Process/Published Artifacts
+
+## Patlet
+
+Teams may be reluctant to use innersource projects that they are unfamiliar with when there is no clear release process apparent in the repository.
```

Comment:
We tend to use InnerSource as anything delivered under the community instead of other naming as innersource, inner source, etc.

Reply:
Fixed.